samples/plugins/postgres_odbc/connectionBuilder.js

```diff
@@ -13,10 +13,20 @@
     params["Fetch"] = "2048";
 
     if (attr[connectionHelper.attributeSSLMode] == "require")
-    {        
+    {
         params["sslmode"] = "require";
     }
 
+    var odbcConnectStringExtrasMap = {};
+    const attributeODBCConnectStringExtras = "odbc-connect-string-extras";
+    if (attributeODBCConnectStringExtras in attr)
+    {
+        odbcConnectStringExtrasMap = connectionHelper.ParseODBCConnectString(attr[attributeODBCConnectStringExtras]);
```

Comment:
This is a 2020.1 feature, so it seems a bit early to update.

Reply:
I'll close this one and open a `hotfix` branch with just my changes.